tests/cypress/integration/remove_users_tasks.js

```diff
@@ -0,0 +1,69 @@
+/*
+ * Copyright (C) 2020 Intel Corporation
+ *
+ * SPDX-License-Identifier: MIT
+ */
+
+/// <reference types="cypress" />
+
+let authKey = ''
+
+describe('Delete users and tasks created during the test run.', () => {
+    it('Get token', () => {
```

Comment:
I would suggest to make this method ``async`` and use ``await`` inside instead of promises.

Reply:
Done. Added method `async` and use `await` for responce.